doc/noobaa-crd.md

```diff
@@ -240,6 +240,22 @@ spec:
   dbConf: |+
     max_connections = 1000
 ```
+## Pod Topology Spread Constraints for Noobaa endpoint
+
+Noobaa operator will add the Topology Spread Constraints to Noobaa endpoint deployment to make sure endpoint pods are evenly spread across nodes. Cluster based on Kubernetes 1.25 and previous versions missing `nodeTaintsPolicy` in `topologySpreadConstraints` and pods are not evenly spread between nodes for these clusters. Noobaa operator checks for the Kubernates version and adds `topologySpreadConstraints` only if the Kubernates version is 1.26+. Operator also skips `topologySpreadConstraints` if this is already set on the deployment, or the CR annotation `noobaa.io/skip_topology_spread_constraints` is set to true.
+
+```yaml
+topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: ScheduleAnyway
+    labelSelector:
+      matchLabels:
+        noobaa-s3: noobaa
+    nodeTaintsPolicy: Honor
+
+```
+Users can make changes to `topologySpreadConstraints` configuration after the operator creates it and the changes will not override it.
```

Comment:
worth clarifying that the default value is resotored if `topologySpreadConstraints` is removed completely